pyspecProcScripts/correlation_alignment.py

```diff
@@ -66,8 +90,12 @@ def correl_align(s, align_phases=False,tol=1e-4,indirect_dim='indirect',fig_titl
     energy_vals.append(this_E / sig_energy)
     last_E = None
     for_nu_center =s.C
-    for_nu_center.ft(['ph1','ph2'])
-    for_nu_center = for_nu_center['ph1',ph1_selection]['ph2',ph2_selection]
+    if ODNP:
+        for_nu_center.ft(['ph1'])
+        for_nu_center = for_nu_center['ph1',ph1_selection]
+    else:
+        for_nu_center.ft(['ph1','ph2'])
+        for_nu_center = for_nu_center['ph1',ph1_selection]['ph2',ph2_selection]
```

Comment:
here, you would would the ft inside a loop over keys and use your `select_pathway` function

Reply:
IDK if this was done *exactly* the way you wanted but I did use the list comprehension and dictionary to do it